mast/models.py

```diff
@@ -1 +1,69 @@
-# ADD DB CLASSES
\ No newline at end of file
+from mast import db, login_manager
+from flask_login import UserMixin
+import enum
```

Comment:
please do not import the whole package, since only one class is used, ergo 
```python
from enum import Enum
```

Reply:
fixed